src/basic/e3a2.MOVCHG.s

```diff
@@ -0,0 +1,36 @@
+;; #LAYOUT# STD *       #TAKE
+;; #LAYOUT# *   BASIC_0 #TAKE
+;; #LAYOUT# *   *       #IGNORE
+
+
+; init - This is the implementation of the CHARGET and CHARGOT functions
+;        The basic cold start will copy this into the zero page
+;
+; Input: Nothing
+; Output: Nothing
+; Clobbers: A, X
+;
+; See also:
+; https://www.c64-wiki.com/wiki/CHRGET
+;
+
+
+MOVCHG:
```

Comment:
This routine won't work with our BASIC implementation, which (depending on the memory model selected in the `src/,,config*` file) can use RAM below the BASIC ROM (or even further...).

If you really won't it, make it available only for the 38K memory model.

Reply:
I don't think I understand what you mean. This routine is copied to the `CHRGET` location in the zero page at cold start. Then the BASIC calls it from there. I put it on this fixed address because there was documentation pointing here. Is this a problem?

Reply:
OK, configuration added:

`make test_testing38` - builds and opens the 38K test binaries in VICE
`make test_testing60` - builds and opens the 60K test binaries in VICE